app/javascript/src/components/Dashboard/Contacts/NewContactForm.jsx

```diff
@@ -1,7 +1,69 @@
 import React from "react";
+import * as yup from "yup";
+import { Formik, Form } from "formik";
+import { Input, Select, Switch } from "neetoui/formik";
+import { Button, Label } from "neetoui";
+import { departmentOptions } from "constants/contacts";
 
-const NewContactForm = () => {
-  return <div>contact form</div>;
-};
-
-export default NewContactForm;
+export default function NewContactForm({ onClose, refetch }) {
+  const handleSubmit = () => {
+    refetch();
+    onClose();
+  };
+  return (
+    <Formik
+      initialValues={{
+        name: "",
+        email: "",
+        department: {},
+        contactNumber: "",
+        addedToBasecamp: false,
+      }}
+      onSubmit={handleSubmit}
+      validationSchema={yup.object({
+        name: yup.string().required("Name is required"),
+        email: yup
+          .string()
+          .email("Invalid email format")
+          .required("Email is required"),
+        contactNumber: yup.string().required("Contact Number is required"),
```

Comment:
Please move `initialValues` and `validationSchema` to a separate constants file.